templates/tools/dockerfile/grpc_artifact_python_manylinux2014_aarch64/Dockerfile.template

```diff
@@ -0,0 +1,28 @@
+%YAML 1.2
+--- |
+  # Copyright 2020 The gRPC Authors
+  #
+  # Licensed under the Apache License, Version 2.0 (the "License");
+  # you may not use this file except in compliance with the License.
+  # You may obtain a copy of the License at
+  #
+  #     http://www.apache.org/licenses/LICENSE-2.0
+  #
+  # Unless required by applicable law or agreed to in writing, software
+  # distributed under the License is distributed on an "AS IS" BASIS,
+  # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  # See the License for the specific language governing permissions and
+  # limitations under the License.
+  
+  FROM dockcross/manylinux2014-aarch64:20230905-7b2d74f
+  
+  # manylinux_2_17 is the preferred alias of manylinux2014
+  ENV AUDITWHEEL_PLAT manylinux_2_17_$AUDITWHEEL_ARCH
+  
+  <%include file="../python_pip_builds.include"/>
+  
+  RUN unset CMAKE_TOOLCHAIN_FILE && unset AS AR CC CPP CXX LD STRIP OBJCOPY
+  <%include file="../ccache_old.include"/>
+  
+  RUN git config --global --add safe.directory /var/local/jenkins/grpc
+  RUN git config --global protocol.file.allow always
```

Comment:
I think we can put those two lines in a file too, something like `git_config.include`?